model/project_aliases.go

```diff
@@ -96,6 +96,16 @@ func (p *ProjectAlias) Upsert() error {
 		variantKey:   p.Variant,
 		tagsKey:      p.Tags,
 	}
+
+	taskSet := p.Task != ""
```

Comment:
The db layer isn't the right place for this sort of validation. This should go in service/project.go. There's also the problem that it's impossible to remove a task regex